src/coreclr/jit/emit.cpp

```diff
@@ -4182,7 +4191,8 @@ void emitter::emitRemoveJumpToNextInst()
             // target group is not bound yet so use the cookie to fetch it
             insGroup* targetGroup = (insGroup*)emitCodeGetCookie(jmp->idAddr()->iiaBBlabel);
 
-            if ((targetGroup != nullptr) && (jmpGroup->igNext == targetGroup))
+            if ((targetGroup != nullptr) && (jmpGroup->igNext == targetGroup) &&
```

Comment:
```suggestion
            if ((jmpGroup->igNext == targetGroup) &&
```

Reply:
And then have `assert(targetGroup != nullptr)` before the if-check? It might be worth knowing if we identified a jump that has no target.

Reply:
Done.